include/shark/Algorithms/Trainers/CSvmTrainer.h

```diff
@@ -42,10 +38,10 @@ enum class McSvm{
 
 
 ///
-/// \brief Training of C-SVMs for binary classification.
+/// \brief Training of C-SVMs for binary classifellation.
```

Comment:
Awesome... I don't know this word, but all results on google are porn :)
This is apparently an artifact of the replacement "ic" -> "ell". It appears multiple times in the document.